packages/by-name/cloud-api-adaptor/package.nix

```diff
@@ -73,6 +73,11 @@ buildGoModule rec {
     default-policy = runCommand "default-policy" { } ''
       cp ${cloud-api-adaptor.src}/src/cloud-api-adaptor/podvm/files/etc/kata-opa/allow-all.rego $out
     '';
+
+    entrypoint = runCommand "entrypoint" { } ''
+      cp ${cloud-api-adaptor.src}/src/cloud-api-adaptor/entrypoint.sh $out
+      chmod +x $out
+    '';
```

Comment:
This does neither properly patch shebangs nor declare the dependencies of the script.